response/core/urls.py

```diff
@@ -1,16 +1,24 @@
 from django.conf.urls import url, include
 from rest_framework import routers
 
-from response.core.views import IncidentViewSet, ActionViewSet, ExternalUserViewSet
+from response.core.views import (
+    IncidentViewSet,
+    IncidentsByMonthViewSet,
+    ActionViewSet,
+    ExternalUserViewSet,
+)
 
 # Routers provide an easy way of automatically determining the URL conf.
 router = routers.DefaultRouter()
-router.register(r'incidents', IncidentViewSet, basename='incident')
-router.register(r'actions', ActionViewSet)
-router.register(r'ExternalUser', ExternalUserViewSet)
+router.register(r"incidents", IncidentViewSet, basename="incident")
+router.register(
+    r"incidents/bymonth/(?P<year>[0-9]{4})/(?P<month>[0-9]{2})",
```

Comment:
Similar to other comment, there's a nicer syntax now. Not required though: https://docs.djangoproject.com/en/2.2/topics/http/urls/#example